starter-code/src/viking.js

```diff
@@ -1,11 +1,94 @@
+'use strict';
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+  this.attack = function (){
+    return this.strength;
+  };
+  this.receiveDamage = function (damage){
+    this.health -= damage;
+  };
+
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+  this.name = name;
+  Soldier.call(this, health, strength);
+  this.receiveDamage = function(damage){
+    this.health -= damage;
+    if(this.health > 0){
+      return this.name + " has received " + damage + " points of damage";
+    }
+    else{
+      return this.name + " has died in act of combat";
+    }
+  };
+  this.battleCry = function(){
+    return "Odin Owns You All!";
+  };
+}
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.contructor = Viking;
```

Comment:
If you use the "simpler" version of oop, you don't need this two lines (32 and 33). If you add your methods with prototype, then you will need them.